src/polyswarm_api/__init__.py

```diff
@@ -408,13 +408,23 @@ def _fix_result(self, result):
         response['search'] = '{hash_type}={hash}'.format(hash_type=hash_type, hash=to_scan)
         return response
 
-    async def search_query(self, query):
+    async def search_query(self, query, raw=True):
         """
         Search by Elasticsearch query using the PS API asynchronously.
 
-        :param query: Elasticsearch query
+        :param query: Elasticsearch JSON query or search string
+        :param raw: If this is a raw JSON query or needs wrapping
```

Comment:
change to be more clear "If True, query is read in as ZYX, else it is read in as AAA"

Reply:
fixed